x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_extension/endpoint_policy_create_extension.tsx

```diff
@@ -245,7 +267,7 @@ export const EndpointPolicyCreateExtension = memo<PackagePolicyCreateExtensionCo
                 <HelpTextWithPadding>
                   <FormattedMessage
                     id="xpack.securitySolution.createPackagePolicy.stepConfigure.packagePolicyTypeEndpointEDREssential"
-                    defaultMessage="Endpoint Alerts, Process Events, Network Events, File Events"
+                    defaultMessage="Everything in NGAV, with file and network telemetry"
```

Comment:
I agree,

Pinging @caitlinbetz to check opinions on that

Reply:
I worry it is too wordy, and will be hard to immediately tell the differences between each? (if we repeat everything from the first option)

Reply:
I think it's only about changing the word "with" to "plus" (As suggested for EDR Complete); So, overall it would be:

**NGAV:**
Machine learning malware, ransomware, memory threat, malicious behavior, and credential theft preventions, plus process telemetry

**EDR Essential**
Everything in NGAV, plus file and network telemetry

**EDR Complete**
Everything in EDR Essential, plus full telemetry

Reply:
oh, got it. that works! thank you